src/components/button-toggle/button-toggle.style.ts

```diff
@@ -65,9 +67,39 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
   box-sizing: border-box;
   max-width: 100%;
 
-  ${({ size }) => css`
+  ${({ variant }) =>
+    variant === "minor" &&
+    css`
+      border: 1px solid var(--colorsActionMinor500);
+
+      &:focus {
+        border: 4px solid var(--colorsActionMinor500);
+        background-color: var(--colorsActionMinor600);
+      }
+
+      &&:hover {
+        color: var(--colorsActionMinorYang100);
+        background-color: var(--colorsActionMinor500);
+        border-color: var(--colorsActionMinor500);
+
+        & + ${StyledButton} {
+          border-left-color: var(--colorsActionMinor500);
+        }
+      }
+
+      & ${StyledIcon} {
+        color: var(--colorsActionMinor500);
+        &&:hover {
+          color: var(--colorsActionMinorYang100);
+        }
+      }
+    `}
+
+  ${({ size, variant }) => css`
     height: ${heightConfig[size]}px;
-    padding: 0 ${paddingConfig[size]}px;
+    padding: ${variant === "default"
+      ? `0 ${paddingLargeIconConfig[size]}px;`
```

Comment:
Where does this come from, I think you can just use `paddingConfig` that's already defined in this file